app/models/qualifications_feature_metrics.rb

```diff
@@ -0,0 +1,51 @@
+class QualificationsFeatureMetrics
+  include ActionView::Helpers::NumberHelper
+
+  def formatted_a_level_percentage(
+    minimum_count,
+    start_time,
+    end_time = Time.zone.now.end_of_day
+  )
+    applications_with_a_levels_count = applications_with_a_levels_grouped(
+      minimum_count,
+      start_time,
+      end_time,
+    ).count.keys.count
```

Comment:
Is this right? Just going to test locally.

Reply:
This is a bit messy. The first count is a SQL `SELECT count(...) ...` for the `GROUP BY` query and returns a hash, the second count is a Ruby `count` on the hash. Maybe I need to refactor this to make it a bit clearer (it confused me).

Reply:
Yea i assumed it was something to with group by. I just couldn't see how the method calls were interacting with the objects in my head.